code/modules/surgery/organs/dismemberment_augment.dm

```diff
@@ -121,6 +121,12 @@
 	update_organ_icon()
 	if(!owner)
 		return 0
+	var/pass = 0
+	for(var/obj/item/organ/internal/X in owner.internal_organs)
+		if(X.zone == "chest")
```

Comment:
Something tells me this will always pass, I think we're should move this new code down to...